tests/test_views.py

```diff
@@ -355,6 +355,37 @@ def test_returns_pong(self):
         self.assertEqual(response['Content-Type'], 'text/plain')
 
 
+class TestBareStatus(unittest.TestCase):
+    def setUp(self):
+        # Ensure that every test executes with separate settings
+        reload_settings()
+
+    def test_bare_status_success(self):
+        request = RequestFactory().get('/')
+        response = views.bare_status(request)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode(), '')
+
+    @patch('watchman.checks._check_databases')
+    @override_settings(WATCHMAN_ERROR_CODE=503)
```

Comment:
Can you add another test that the default error code for the bare status view will be `500`? You can just copy this one without the override.

Reply:
Fixed in 657f135